ipatests/test_xmlrpc/test_automember_plugin.py

```diff
@@ -528,6 +689,171 @@ def test_rebuild_membership_hosts_group(self, automember_hostgroup, user1,
             command()
 
 
+class TestAutomemberAddConditionNegative(XMLRPC_test):
+
+    @pytest.mark.parametrize(
+        "automember_grp, grp", [
+            ("automember_group", "group1"),
+            ("automember_hostgroup", "hostgroup1")
+        ])
+    def test_create_inclusive_with_nonexistent_group(self, automember_grp,
+                                                     grp, request):
+        grp = request.getfixturevalue(grp)
```

Comment:
The function can be simplified and use only automember_grp + request parameters (grp is not useful here, grp.cn can be replaced with automember_grp.cn).